js/plates-interactions/globe-interactions-manager.ts

```diff
@@ -27,6 +27,9 @@ export default class GlobeInteractionsManager extends BaseInteractionsManager {
       markField: new PlanetClick({ ...baseOptions, startEventName: "markField" }),
       continentDrawing: new PlanetClick({ ...baseOptions, startEventName: "continentDrawing", moveEventName: "continentDrawing", endEventName: "continentDrawingEnd" }),
       continentErasing: new PlanetClick({ ...baseOptions, startEventName: "continentErasing", moveEventName: "continentErasing", endEventName: "continentErasingEnd" }),
+      measureTempPressure: new PlanetClick({
+        ...baseOptions, cursor: "not-allowed", startEventName: "measureTempPressure",
+        moveEventName: "measureTempPressure", endEventName: "measureTempPressureEnd" }),
```

Comment:
Do we want to specify events here? It seems like they'll be emitted even though we don't support the temp-pressure tool in the globe view?

Reply:
I was just following existing patterns. If they're not required then I'll remove them.

Reply:
That's the globe interactions manager, so I guess we won't have to deal with any events here, as this tool will not work with the globe view. I guess the only reason to keep this line is to set `cursor: "not-allowed` if we can't do it using plain CSS somewhere else.

Reply:
CSS doesn't work because the cursor is set via inline style from the current interaction, so defining a globe interaction for setting the cursor seemed like the path of least resistance.